main/store/state/types/frames.ts

```diff
@@ -2,7 +2,7 @@ import { z } from 'zod'
 
 const ViewSchema = z.object({
   id: z.string(),
-  ready: z.boolean(),
+  // ready: z.boolean(),
```

Comment:
should we just get rid of this?

Reply:
need to restore the ready state we have `workspacesMeta` or somewhere else in the state for it to go